apps/hpl/2.1/metadata.yml

```diff
@@ -46,48 +48,29 @@
     - mpi/openmpi
     - libs/atlas
 :sources:
-  - 4node_hpl.alces.dat
+  - 8proc_hpl.alces.dat
+  - 8proc_hpl_example.sh.tpl
```

Comment:
Can we use `-` rather than `_` please? i.e. `8proc-hpl-example`.  Come to think of it, we probably don't need `-example` as a suffix.  I think we should just make it `8proc-hpl`.

Or perhaps `hpl-8proc`? Then all `hpl`-related scripts will be together if sorted lexographically.